cli/src/templater.rs

```diff
@@ -184,6 +185,17 @@ where
     }
 }
 
+pub struct RawEscapeSequenceTemplate<T>(pub T);
+
+impl<T: Template> Template for RawEscapeSequenceTemplate<T> {
+    fn format(&self, formatter: &mut TemplateFormatter) -> io::Result<()> {
+        let rewrap = formatter.rewrap_fn();
+        let mut raw_escape_sequence_formatter = PlainTextFormatter::new(formatter.raw());
```

Comment:
nit: I would rename it to `raw_formatter` (because "escape_sequence" isn't important in this local context.)